malcolm/core/numbermeta.py

```diff
@@ -0,0 +1,40 @@
+from malcolm.core.attributemeta import AttributeMeta
+
+from pkg_resources import require
+require("numpy")
+import numpy as np
+
+
+_metas = {
```

Comment:
(matter of taste) I probably wouldn't use an intermediate dictionary, I think it's probably easier to read if you just put the strings straight into the decorator calls...